src/library/scala/collection/GenMapLike.scala

```diff
@@ -125,7 +125,7 @@ trait GenMapLike[A, +B, +Repr] extends GenIterableLike[(A, B), Repr] with Equals
         }
       } catch {
         case ex: ClassCastException =>
```

Comment:
I wasn't sure what the coding style is. Only in this method, there are two other `case`s with potential one-liners where one is on a single line and the other isn't ^^
I can change it, sure.